mysql/src/main/kotlin/com/huanshankeji/exposedvertxsqlclient/mysql/DatabaseClient.kt

```diff
@@ -0,0 +1,9 @@
+package com.huanshankeji.exposedvertxsqlclient.mysql
+
+import com.huanshankeji.exposedvertxsqlclient.DatabaseClient
+import com.huanshankeji.exposedvertxsqlclient.withTypedTransaction
+import io.vertx.sqlclient.Pool
+import io.vertx.sqlclient.SqlConnection
+
+suspend fun <T> DatabaseClient<Pool>.withSQLTransaction(function: suspend (DatabaseClient<SqlConnection>) -> T): T =
```

Comment:
This one should be `withMysqlTransaction` and the connection type should be `MySQLConnection`.